src/domain/event/hooks/useEventFieldsData.ts

```diff
@@ -0,0 +1,33 @@
+import { INCLUDE, KEYWORD_SETS } from '../../../constants';
+import {
+  useKeywordSetQuery,
+  useLanguagesQuery,
+} from '../../../generated/graphql';
+import getPathBuilder from '../../../utils/getPathBuilder';
+import { keywordSetPathBuilder } from '../../keywordSet/utils';
+
+const useEventFieldsData = () => {
+  const { loading: loadingLanguages } = useLanguagesQuery();
+
+  const { loading: loadingKeywords } = useKeywordSetQuery({
+    variables: {
+      createPath: getPathBuilder(keywordSetPathBuilder),
+      id: KEYWORD_SETS.TOPICS,
+      include: [INCLUDE.KEYWORDS],
+    },
+  });
+
+  const { loading: loadingAudiences } = useKeywordSetQuery({
+    variables: {
+      createPath: getPathBuilder(keywordSetPathBuilder),
+      id: KEYWORD_SETS.AUDIENCES,
+      include: [INCLUDE.KEYWORDS],
+    },
+  });
+
+  const loading = loadingLanguages || loadingKeywords || loadingAudiences;
+
+  return { loading };
```

Comment:
Is there a reason to return object instead of just loading?

Reply:
Added audienceData and topics data to return value and started using it on audience section and classification section